src/screens/WhotoFollow/index.js

```diff
@@ -22,7 +22,6 @@ import Label from './elements/Label';
 import Loading from '../Loading';
 import TokenStorage from '../../utils/storage/custom/tokenStorage';
 import dimen from '../../utils/dimen';
-import useProfileHook from '../../hooks/core/profile/useProfileHook';
 import {Analytics} from '../../libraries/analytics/firebaseAnalytics';
 import {Button} from '../../components/Button';
 import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useProfileHook` import has been removed. If this hook was used previously in the component, ensure that its functionality is replaced appropriately.
```diff
- import useProfileHook from '../../hooks/core/profile/useProfileHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->